modules/microshift-configuring-ovn.adoc

```diff
@@ -71,4 +70,11 @@ mtu: 1300
 ----
 
 [IMPORTANT]
+====
 When `disableOVSInit` is set to true in the `ovn.yaml` config file, the OVS bridge br-ex must be manually configured.
+====
+
+[IMPORTANT]
+====
+If you change the `mtu` configuration value in the `ovn.yaml` file, you must restart the node for the updated setting to apply.
```

Comment:
It would be more clear to say, "you must 'reboot' the host on which MicroShift is running".